testutil/ciao-down/ciao_down.go

```diff
@@ -117,28 +117,32 @@ func downloadProgress(p progress) {
 }
 
 func prepare(ctx context.Context, errCh chan error) {
+	var err error
+
+	defer func() {
+		errCh <- err
```

Comment:
Currently, we only send error down the channel so there's no need to close it.  The reason we use this model is so we can multiplex the result of the command with the CTRL-C signal.